src/app/(main)/EventPage.tsx

```diff
@@ -101,19 +101,25 @@ export const EventPage = ({ eventId }: { eventId: string }) => {
     const isActivityActive = isActive(activity)
     body = (
       <Box>
-        <Typography variant="h4">{activity.title}</Typography>
+        <Stack direction="row" spacing={1} sx={{mt:2, mb:2}}>
+          <Typography variant="h4" sx={{mr:"auto"}}>{activity.title}</Typography>
+          
+          <Stack direction="row" spacing={1} sx={{height:"2.125rem"}}>

```

Comment:
We can use flex-box `align-items` to position the buttons without setting a specific size.
We can also use MUI breakpoints to change the stack direction from `column` to `row` for narrow screens.

```suggestion
        <Stack direction={{ xs: 'column', sm: 'row' }} spacing={1} alignItems="start" sx={{mt:2, mb:2}}>
          <Typography variant="h4" flexGrow={1}>{activity.title}</Typography>
          
          <Stack direction="row" spacing={1} alignItems="center">
```

![image](https://github.com/KingCountySAR/respond-next/assets/1082784/c8861d73-90ec-4fa7-9e19-87774fcec746)

![image](https://github.com/KingCountySAR/respond-next/assets/1082784/36dcf1bf-e39a-46ef-9fc4-9289f8f0702c)

Reply:
Very cool! Didn't realize these were all flex boxes. Happy to go with that.